src/scores/AnnotationColumn.ts

```diff
@@ -33,9 +33,9 @@ export class AnnotationColumn extends ABooleanScore implements IScore<number> {
   protected get columnName() {
     return 'namedset_containment';
   }
-  static createAnnotationColumnScore(data, pluginDesc: IPluginDesc) {
+  static createAnnotationColumnScore(data: IAnnotationColumnParam[], pluginDesc: IPluginDesc) {
     const {primary} = ScoreUtils.selectDataSources(pluginDesc);
-    return new AnnotationColumn(data, primary);
+    return data.map((d) => new AnnotationColumn(d, primary));
```

Comment:
This does not work with existing provenance graphs and causes errors!

![grafik](https://user-images.githubusercontent.com/5851088/95744449-984f6d00-0c93-11eb-88ea-f69bcfdc1545.png)

Please check if `data` is an array and handle both cases (single data and data array).